prowler/providers/aws/services/rds/rds_service.py

```diff
@@ -298,6 +251,51 @@ def __describe_db_clusters__(self, regional_client):
                 f"{regional_client.region} -- {error.__class__.__name__}[{error.__traceback__.tb_lineno}]: {error}"
             )
 
+    def __describe_db_cluster_parameters__(self, regional_client):
+        logger.info("RDS - Describe DB Cluster Parameters...")
+        try:
+            for cluster in self.db_clusters.values():
+                if cluster.region == regional_client.region:
+                    try:
+                        describe_db_cluster_parameters_paginator = (
+                            regional_client.get_paginator(
+                                "describe_db_cluster_parameters"
+                            )
+                        )
+                        for page in describe_db_cluster_parameters_paginator.paginate(
+                            DBClusterParameterGroupName=cluster.parameter_group
+                        ):
+                            for parameter in page["Parameters"]:
```

Comment:
Do we need to include two `try/except` here? One for each loop?

Reply:
I don't think it is necessary, what we usually do is one try for the whole function and another one for each resource

Reply:
yes but if something breaks within the loop we are not going to loop over and we should.